Middlewares/GlobalExceptionHandlerMiddleware.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Azure.Functions.Worker.Middleware;
+using Microsoft.Extensions.Logging;
+using System.Net;
+
+namespace Middlewares
+{
+    public class GlobalExceptionHandlerMiddleware : IFunctionsWorkerMiddleware
+    {
+        private readonly ILogger<GlobalExceptionHandlerMiddleware> _logger;
+
+        public GlobalExceptionHandlerMiddleware(ILogger<GlobalExceptionHandlerMiddleware> logger)
+        {
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        }
+        public async Task Invoke(FunctionContext context, FunctionExecutionDelegate next)
+        {
+			try
+			{
+				await next(context);
+			}
+			catch (Exception ex)
```

Comment:
Can we do an example of a custom exception, let's call it `InvalidInputException` for now, and we catch this exception, then we return a 400 with the exception message please? 

If it's any other type of exception (at the time being) throw a 500 just like we do already